src/txdb.h

```diff
@@ -22,11 +22,17 @@ class CCoinsViewDBCursor;
 class uint256;
 
 //! -dbcache default (MiB)
-static const int64_t nDefaultDbCache = 100;
+static const int64_t nDefaultDbCache = 300;
 //! max. -dbcache in (MiB)
 static const int64_t nMaxDbCache = sizeof(void*) > 4 ? 16384 : 1024;
 //! min. -dbcache in (MiB)
 static const int64_t nMinDbCache = 4;
+//! Max memory allocated to block tree DB specific cache (if no -txindex)
+static const int64_t nMaxBlockDBCache = 2 << 20;
+//! Max memory allocated to block tree DB specific cache (with -txindex)
+static const int64_t nMaxBlockDBAndTxIndexCache = 32 << 20;
+//! Max memory allocated to coin DB specific cache
+static const int64_t nMaxCoinsDBCache = 32 << 20;
```

Comment:
yes, I don't want to mention numbers in the comments (as everyone forgets to update those) but specifying that it's MiB is fine with me